rocky_8/kickstart.ks

```diff
@@ -0,0 +1,53 @@
+#auth --enableshadow --passalgo=sha512 --kickstart
+# For some reason libguest can't detect the default (xfs)
+# file system for CentOS 7 installs;  Overridden to use
+# ext4 - Moate
+autopart --type=lvm --fstype=ext4
```

Comment:
Do you know if we still need this for EL8?

Reply:
No idea, never tried. The CentOS docs say that 8 also uses XFS as the default file system so unless libguest has been fixed, we'll need it there too.

Reply:
We don't appear to need it for 8.